registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/test/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessorTest.java

```diff
@@ -450,11 +456,11 @@ public void testBioDeDupUpdatePacketHandlerProcessingSuccess() throws ApisResour
 	 * @throws RegistrationProcessorCheckedException
 	 */
 	@Test
-	@Ignore
```

Comment:
Shall we remove this test case or just keep @Ignore annotation.

Reply:
done